api_helper.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import logging
+
+class APIFetch:
+    def __init__(self) -> None:
+        pass
+    def fetch_data(self,url,country:None):
+        if country is not None:
```

Comment:
in this IF and else, most of the code is repetative. can you reduce this more
I think IF and else should have only modificaction  of the URL. rest everything is same